App/SplitViewController.swift

```diff
@@ -128,7 +128,7 @@ class SplitViewController: UISplitViewController {
             let controller = UIHostingController(rootView: Settings())
             setViewController(UINavigationController(rootViewController: controller), for: .secondary)
         default:
-            let controller = UIHostingController(rootView: Text("Not yet implemented".localized))
```

Comment:
I am not sure if this part needs localisation at all... The user should never see this case.